docs/index.md

```diff
@@ -11,6 +11,37 @@ UP4W can be installed from the [Microsoft Store](https://apps.microsoft.com/home
 
 WSL is preferred by many organisations as a solution to run a fully-functioning Linux environment on a Windows machine. UP4W empowers system administrators and corporate security teams to manage these WSL instances at scale.
 
+## In this documentation
+
+````{grid} 1 1 2 2
+
+```{grid-item-card} [Tutorial](tutorial/index)
+
+**Start here** with a hands-on introduction for new users, guiding you through your first-steps
+```
+
+```{grid-item-card} [How-to guides](howto/index)
+
+**Follow step-by-step** instructions for key operations and common tasks
+```
+
+````
+
+````{grid} 1 1 2 2
+:reverse:
+
+```{grid-item-card} [Reference](reference/index)
+
+**Read technical descriptions** of important factual information relating to UP4W
+```
+
+```{grid-item-card} [UP4W Dev](dev/index)
+
+**Review guides and reference material** aimed at developers
```

Comment:
We should clarify that by developers we mean contributors to this project.

Reply:
Thanks Carlos -- good point. I will change to "aimed at contributors". For now, I think we should keep "UP4W Dev" for that section of the docs. Ultimately, the docs for contributors may not need to be in a separate place; if they are clearly labelled and articulated they should be able to go alongside the regular docs but that's another conversation for another day.